arch/arm/src/stm32l4/stm32l4_adc.c

```diff
@@ -1291,10 +1331,11 @@ static int adc_setup(FAR struct adc_dev_s *dev)
   adc_putreg(priv, STM32L4_ADC_SMPR1_OFFSET, ADC_SMPR1_DEFAULT);
   adc_putreg(priv, STM32L4_ADC_SMPR2_OFFSET, ADC_SMPR2_DEFAULT);
 
+  #ifdef HAVE_ADC_RESOLUTION
```

Comment:
Is this line formatted correctly?

Reply:
@ghn-certi Which line are you talking about? Are you referring to the coding standard?

Thanks

Reply:
Hi @daniel-p-carvalho, I think @ghn-certi is talking about the "#ifdef" line, normally all #ifdef lines starts totally at left size even for indented code.